tests/test_diffraction_objects.py

```diff
@@ -8,43 +8,6 @@
 
 from diffpy.utils.diffraction_objects import XQUANTITIES, DiffractionObject
 
-
-def compare_dicts(dict1, dict2):
```

Comment:
These functions are no longer used in our test code - We. may use the `DeepDiff` library instead to compare two dict as implemented in our previous PR:

```python
@pytest.mark.parametrize("inputs, expected", tc_params)
def test_constructor(inputs, expected):
    actual_do = DiffractionObject(**inputs)
    diff = DeepDiff(actual_do.__dict__, expected, ignore_order=True, significant_digits=13)
    assert diff == {}
```